### DISCUSSION TITLE
community promotion

### DISCUSSION THREAD

**Discussion by majidaldo** [pythological/kanren#61] [General]
do you have any ideas for how to promote this library? i can't help but think that i can solve so many problems if i just encode them logically/symbolically. 

however, usually the toy example problems (map coloring problem, parent/child 'queries') are a far cry from practical application.

**Comment by rlouf** [comment]
I started to work on a relational implementation of https://github.com/mxgmn/WaveFunctionCollapse, but quickly ran into #59. It's would be a cool application, on top of stress-testing the library's performance.

**Reply by brandonwillard** [reply]
Yeah, that's a good issue to look into.  There are also some basic performance improvements we need to implement from the Scheme miniKanren implementations, as well as the basic literature (e.g. tabling).
